.github/workflows/build.yml

```diff
@@ -14,14 +14,20 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
Set `ubuntu-latest` to 22.04 or 24.04 so that GH Actions doesn't alert repo owner about sudden OS change.

Reply:
will be done after this pull request as minegame will be reworking the publishing process